### DISCUSSION TITLE
Need to be able to book multiple timeslots

### DISCUSSION THREAD

**Discussion by mimiyin** [ITPNYU/booking-app#61] [General]
Recurring times but also a random selection of dates/times.

**Comment by nopivnick** [comment]
so this is a UI issue. given our UI shows availability one day at a time, how do we represent the presence of a conflict on a date that's not currently selected?

my guess is other than our interdepartmental pre-semester jockeying for space, which uses a different process (Google Sheet) anyway, both recurring and multiple dates are edge cases.
